diff -Nur -x '*.orig' -x '*~' vrjuggler-2.2-gitsvn/modules/tweek/java/Makefile.in vrjuggler-2.2-gitsvn.new/modules/tweek/java/Makefile.in
--- vrjuggler-2.2-gitsvn/modules/tweek/java/Makefile.in	2010-11-15 14:14:57.835721755 -0600
+++ vrjuggler-2.2-gitsvn.new/modules/tweek/java/Makefile.in	2010-11-15 14:20:21.606734387 -0600
@@ -161,10 +161,10 @@
 ifeq (@OS_TYPE@, UNIX)
 	$(INSTALL) $(EXTRA_INSTALL_ARGS) -m $(EXEC_PERMS)		\
           $(GROUP_OPT_UNIX) $(topdir)/java/tweek.sh			\
-          $(bindir)/tweek
+          $(bindir)/tweek-1.2
 	$(INSTALL) $(EXTRA_INSTALL_ARGS) -m $(FILE_PERMS)		\
           $(GROUP_OPT_UNIX) $(topdir)/java/tweek-base.sh		\
-          $(bindir)/tweek-base.sh
+          $(bindir)/tweek-base-1.2.sh
 else
 	$(INSTALL) -m $(EXEC_PERMS) $(GROUP_OPT_UNIX)			\
           $(TWEEKROOT_ABS)/java/tweek.bat $(bindir)/tweek.bat
diff -Nur -x '*.orig' -x '*~' vrjuggler-2.2-gitsvn/modules/tweek/java/tweek.sh.in vrjuggler-2.2-gitsvn.new/modules/tweek/java/tweek.sh.in
--- vrjuggler-2.2-gitsvn/modules/tweek/java/tweek.sh.in	2010-11-15 14:14:58.387988606 -0600
+++ vrjuggler-2.2-gitsvn.new/modules/tweek/java/tweek.sh.in	2010-11-15 14:19:15.494805067 -0600
@@ -70,6 +70,6 @@
 
 EXTRA_JDK_ARGS="-Dsun.java2d.opengl=true"
 
-. $TWEEK_BASE_DIR/bin/tweek-base.sh
+. $TWEEK_BASE_DIR/bin/tweek-base-1.2.sh
 
 exit $status
diff -Nur -x '*.orig' -x '*~' vrjuggler-2.2-gitsvn/modules/vrjuggler/vrjconfig/Makefile.in vrjuggler-2.2-gitsvn.new/modules/vrjuggler/vrjconfig/Makefile.in
--- vrjuggler-2.2-gitsvn/modules/vrjuggler/vrjconfig/Makefile.in	2010-11-15 14:14:58.206739878 -0600
+++ vrjuggler-2.2-gitsvn.new/modules/vrjuggler/vrjconfig/Makefile.in	2010-11-15 14:22:06.615486881 -0600
@@ -116,7 +116,7 @@
 	@echo "------------------------------------------------"
 ifeq (@OS_TYPE@, UNIX)
 	$(INSTALL) -m $(EXEC_PERMS) $(GROUP_OPT_UNIX)			\
-          $(EXTRA_INSTALL_ARGS) $(topdir)/vrjconfig.sh $(bindir)/vrjconfig
+          $(EXTRA_INSTALL_ARGS) $(topdir)/vrjconfig.sh $(bindir)/vrjconfig-2.2
 else
 	$(INSTALL) -m $(EXEC_PERMS) $(GROUP_OPT_UNIX)			\
           $(VJROOT_ABS)/vrjconfig.bat $(bindir)/vrjconfig.bat
diff -Nur -x '*.orig' -x '*~' vrjuggler-2.2-gitsvn/modules/vrjuggler/vrjconfig/vrjconfig.sh.in vrjuggler-2.2-gitsvn.new/modules/vrjuggler/vrjconfig/vrjconfig.sh.in
--- vrjuggler-2.2-gitsvn/modules/vrjuggler/vrjconfig/vrjconfig.sh.in	2010-11-15 14:14:58.387988606 -0600
+++ vrjuggler-2.2-gitsvn.new/modules/vrjuggler/vrjconfig/vrjconfig.sh.in	2010-11-15 14:21:30.244677589 -0600
@@ -74,7 +74,7 @@
 
 APP_NAME='VRJConfig'
 
-. $TWEEK_BASE_DIR/bin/tweek-base.sh \
+. $TWEEK_BASE_DIR/bin/tweek-base-1.2.sh \
    --beanpath="$JCCL_BASE_DIR/@JCCL_DATA_DIR@/beans" \
    --beanpath="$VJ_BASE_DIR/@VJ_DATA_DIR@/beans" \
    --defaultbean="Configuration Editor" "$@"
